tasks.py

```diff
@@ -627,7 +627,7 @@ def hadolint(context):
 @task
 def markdownlint(context):
     """Lint Markdown files."""
-    command = "markdownlint --ignore nautobot/project-static --ignore nautobot/ui/node_modules --config .markdownlint.yml --rules scripts/use-relative-md-links.js nautobot examples *.md"
+    command = "npm exec -- markdownlint --ignore nautobot/project-static --ignore nautobot/ui/node_modules --config .markdownlint.yml --rules scripts/use-relative-md-links.js nautobot examples *.md"
```

Comment:
Just for my own knowledge can we not use `npx` here?

Reply:
From my finding `npx` can sometimes install the utility for you. In this case I want to ensure the version provided in `package.json` is used or fail otherwise (since I don't want to run mismatched versions of markdownlint). Might be preferred to use `npm exec --` or `npx --no-install` in all cases.